docs/consensus/overview.md

```diff
@@ -7,20 +7,75 @@ keywords:
     - Dilithium
 ---
 
-Subspace is powered by *Dilithium* - a lightweight and secure consensus mechanism that is environmentally friendly, permissionless, and fair. *Dilithium* is a Proof-of-Archival-Storage (PoAS), Nakamoto-style consensus protocol based on proofs of storing the blockchain's history. 
+Subspace is powered by *Dilithium* - a lightweight and secure consensus mechanism that is environmentally friendly, permissionless, and fair. *Dilithium* is a Proof-of-Archival-Storage (PoAS), Nakamoto-style consensus protocol based on proofs of storing the blockchain history. 
 
-In PoAS, farmers (not miners) store as many unique pieces of the blockchain history as their disk space allows. PoAS incentivizes the storage of the blockchain history, resolving the critical mechanism design failure, which has hindered scalability and led to centralization within Proof-of-Space blockchains like Filecoin and Chia. Since PoAS consensus is based on storage rather than computing power or wealth, it is eco-friendly and accessible to ordinary people with available disk space. This approach allows us to combine the high security of Bitcoin-style Proof-of-Work with the energy efficiency of Ethereum-style Proof-of-Stake. PoAS also provides the basis for a more generic solution to the blockchain bloat problem. 
+*Dilithium* is a second-generation PoAS consensus algorithm that uses erasure coding and KZG commitments for distributed archiving while combining polynomial encoding with an ASIC-resistant Proof-of-Space for plotting and AES-based Proof-of-Time. The protocol represents a significant step forward in security and user experience for Subspace Network participants. *Dilithium* is also designed to be SSD-friendly, further enhancing energy efficiency and decentralization. 
+
+For those familiar with our initial consensus design, *Dilithium* fulfills all the fundamental ideas described in the original [whitepaper](https://subspace.network/news/subspace-network-whitepaper) but implements them better.
+
+In PoAS, farmers (not miners) store as many unique pieces of the blockchain history as their disk space allows. PoAS incentivizes the storage of the blockchain history, resolving the key mechanism design failure which has hindered scalability and led to centralization within Proof-of-Storage blockchains like Filecoin and Chia. Since PoAS consensus is based on storage, rather than compute power or wealth, it is eco-friendly while remaining accessible to ordinary people with available disk space. This allows it to combine the high security of Bitcoin-style Proof-of-Work with the energy-efficiency of Ethereum-style Proof-of-Stake. It also turns out that PoAS provides the basis for a more generic solution to the problem of blockchain bloat. 
```

Comment:
>Proof-of-Storage

In other places we write Proof-of-Space. I think "storage" is good, because of PoAS, but we need to be consistent.

Reply:
In most places we use PoSpace  to refer to Chia's component or some abstract protocol. Here I used storage to contrast with PoAS and because it mentions filecoin too

Reply:
See, for example, [here](https://github.com/subspace/subnomicon/pull/8/files#diff-678c16972240f0c751f7446c372e85a58918009d926563c086a8506c2e4e4a62R10) or [here](https://github.com/subspace/subnomicon/pull/8/files#diff-678c16972240f0c751f7446c372e85a58918009d926563c086a8506c2e4e4a62R24).